src/pages/creating-challenge/component.js

```diff
@@ -0,0 +1,93 @@
+import React, {useState} from "react";
+import Header from "../../components/header";
+import {Link} from "react-router-dom";
+import CNST from "../../constants";
+import closeIcon from "../../images/close-white.svg";
+import styles from "./_.module.css";
+import {Button, Input, Switch, notification} from "antd";
+import userImg from "../../images/user.png";
+import Settings from "../../components/settings";
+import challenges from "../../images/challenges.svg";
+import VideoCamera from "../../images/video-camera.svg";
+import friendsIcon from "../../images/my-friends.svg";
+import {Helmet} from "react-helmet";
+const {TextArea} = Input;
+
+const openNotification = () => {
+  notification.info({
+    message: `It is not a part of the prototype`,
+    placement: "topLeft",
+  });
+};
+
+const SettingsIcon = ({className}) => (
+  <Link to={CNST.ROUTES.DASHBOARD}>
+    <img src={closeIcon} alt="Go back" className={className} />
+  </Link>
+);
+
+const UserImage = ({userImg, className}) => (
+  <img src={userImg} alt="User" className={className} />
+);
+
+export const CreateChallenge = () => {
+  const [isChecked, handleChange] = useState(false);
```

Comment:
Maybe it would be better to rename `handleChanged` to something like `setIsChecked` or `setChecked` to avoid future confusion.

Reply:
done